content-repo/gen_top_contrib.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import re
+
+import requests
+import urllib3
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
```

Comment:
please override `print` with `timestamped_print` as we do in the other python scripts

Reply:
Don't think it is necessary here, we don't care about the timestamps here. 
We use the print only for 404 not found. the time here is not important.

Reply:
it helps when debugging in the build console, lets add it

Reply:
Done